messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/deployment/injection/InjectedJMSContext.java

```diff
@@ -141,4 +154,25 @@ private Object lookup(String name) {
             }
         }
     }
-}
+
+    /**
+     * Synchronization task, which executes "cleanup" on JMSContext in AfterCompletion phase. BeforeCompletion does nothing.
+     */
+    private class AfterCompletionSynchronization implements Synchronization {
+        protected final Runnable cleanUpMethod;
+
+        public AfterCompletionSynchronization(AbstractJMSContext context) {
+            this.cleanUpMethod = context.getCleanUpMethod();
+        }
+
+        @Override
+        public void beforeCompletion() {
+            //intentionally blank
+        }
+
+        @Override
+        public void afterCompletion(int status) {
+            //intentionally blank
```

Comment:
no longer intentionally blank ;)

Reply:
I agree:) but I've seen this approach in other pieces of code and it could help new developers (like me) understand that there is no missing piece of code. Should I remove it?

Reply:
yes as you override the method and add code in it.